net/BUILD.gn

```diff
@@ -1,10 +1,19 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
 import("//brave/components/tor/buildflags/buildflags.gni")
 
+source_set("net") {
```

Comment:
this should not exist.

Reply:
This makes sense now, based on your other comment: https://github.com/brave/brave-core/pull/16217#discussion_r1087500282